src/components/modules/blockManager.ts

```diff
@@ -873,7 +873,7 @@ export default class BlockManager extends Module {
    * Sets current Block Index -1 which means unknown
    * and clear highlights
    */
-  public dropPointer(): void {
+  public unsetCurrentBlock(): void {
```

Comment:
maybe `clearCurrentBlock()`?

Reply:
"clear" means that block will became empty, but we're not changing the block itself